packages/components/src/components/select/select.lite.tsx

```diff
@@ -94,7 +94,7 @@ export default function DBSelect(props: DBSelectProps) {
 				}
 			} else if (stringPropVisible(props.message, props.showMessage)) {
 				state._descByIds = state._messageId;
-			} else {
+			} else if (props.placeholder) {
 				state._descByIds = state._placeholderId;
 			}
```

Comment:
```suggestion
			} else if (props.placeholder) {
				state._descByIds = state._placeholderId;
			} else {
				state._descByIds = undefined;
			}
```

We would still need a default case.